great_expectations/core/factory/checkpoint_factory.py

```diff
@@ -23,6 +23,10 @@
 
 @public_api
 class CheckpointFactory(Factory[Checkpoint]):
+    """
+    Responsible for managing CRUD operations for Checkpoints.
+    """
+
```

Comment:
These show up in docs without any explanation - we should ensure that anything with `@public_api` has a docstring